src/render/direct3d/SDL_render_d3d.c

```diff
@@ -1553,6 +1554,20 @@ D3D_DestroyTexture(SDL_Renderer * renderer, SDL_Texture * texture)
     texture->driverdata = NULL;
 }
 
+static void
+D3D_UpdateVSync(void * userdata, const char * name, const char * oldValue, const char * newValue)
+{
+    SDL_Renderer *renderer = userdata;
+    D3D_RenderData *data = renderer->driverdata;
+    if (SDL_GetStringBoolean(newValue, SDL_FALSE)) {
+        data->pparams.PresentationInterval = D3DPRESENT_INTERVAL_ONE;
+        renderer->info.flags |= SDL_RENDERER_PRESENTVSYNC;
+    } else {
+        data->pparams.PresentationInterval = D3DPRESENT_INTERVAL_IMMEDIATE;
+        renderer->info.flags &= ~SDL_RENDERER_PRESENTVSYNC;
+    }
```

Comment:
This needs to call D3D_Reset so that the pparams update actually gets sent to the device.

Reply:
Fixed, rebased, and force-pushed.